frontend/src/components/rankingTeamList.js

```diff
@@ -152,6 +152,13 @@ class RankingTeamList extends Component {
             <div className="header">
               <h4 className="title">Rankings</h4>
             </div>
+            {this.props.episode_info.frozen && (
+              <div className="content">
+                Scrimmages may not be requested, due to a submission freeze for
```

Comment:
Do we only want to say this when `canRequest` is true (since people can't request from the rankings page anyway?)